index.js

```diff
@@ -193,12 +189,13 @@ export default class Carousel extends Component {
     }
   }
 
-  _scrollTo = (offset, animated) => {
+  _scrollTo = (offset, animated, options) => {
```

Comment:
Maybe if we introduce `options` it makes sense to move offset and animated there too?
smth. like:

```js
_scrollTo = ({offset, animated, nofix}) => {
  //code
}
```

or just

```js
_scrollTo = (offset, animated, {nofix}) => {
  //code
}
```

in both cases we don't need this line anymore (profit!):
```js
const nofix = options && (options.nofix || false);
```